### PR TITLE
RHDEVDOCS 5984 disabling inline spec

### PR DESCRIPTION
**Pull request by mramendi**
Version(s):

pipeline-docs-main only

Issue:

RHDEVDOCS 5984

Link to docs preview:

https://75534--ocpdocs-pr.netlify.app/openshift-pipelines/latest/install_config/customizing-configurations-in-the-tektonconfig-cr.html#op-disabling-inline-spec_customizing-configurations-in-the-tektonconfig-cr

QE review:
- [x] QE has approved this change.

Additional information:

NOTE TO PEER/MERGE REVIEWERS: the added module uses a concept format and not a procedure format in order to preserve uniformity with other modules in the assembly. Adding a full procedure to edit the TektonConfig CR to every module would be repetitive and would also make for a very large PR. See the preview. or else https://docs.openshift.com/pipelines/1.14/install_config/customizing-configurations-in-the-tektonconfig-cr.html#op-changing-default-service-account_customizing-configurations-in-the-tektonconfig-cr in the current documentation, for several modules that immediately precede this module

### CODE REVIEW

**Comment by mramendi** [comment]
https://issues.redhat.com/browse/RHDEVDOCS-5984

**Review by khrm — COMMENTED** [review]
LGTM

**Review by khrm — APPROVED** [review]

File: modules/op-disabling-inline-spec.adoc [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+
+.Supported values for the `disable-inline-spec` parameter
+|===
+| Value | Description
+
+|`pipeline`
+|You cannot use a `taskSpec:` spec to define a task inside a `Pipeline` CR. Instead, you must use a `taskRef:` spec to incorporate a task from a `Task` CR or to specify a task using a resolver.
+
+|`pipelinerun`
+|You cannot use a `pipelineSpec:` spec to define a pipeline inside a `PipelineRun` CR. Instead, you must use a `pipelineRef:` spec to incorporate a pipeline from a `Pipeline` CR or to specify a pipeline using a resolver.
```
Comment (ppitonak) [inline modules/op-disabling-inline-spec.adoc:80]: @khrm what happens when I set `disable-inline-spec: "pipeline"` ... will I be able to use `taskSpec` inside `pipelinerun`?

**Review by ppitonak — APPROVED** [review]

File: modules/op-disabling-inline-spec.adoc [hunk 2]
```diff
@@ -0,0 +1,85 @@
+// This module is included in the following assemblies:
+// * install_config/customizing-configurations-in-the-tektonconfig-cr.adoc
+
+:_mod-docs-content-type: CONCEPT
+[id="op-disabling-inline-spec_{context}"]
+= Disabling inline specification of pipelines and tasks
+
+By default, {pipelines-shortname} supports inline specifification of pipelines and tasks in the following cases:
```
Comment (adellape) [inline modules/op-disabling-inline-spec.adoc:8]: Typo:
```suggestion
By default, {pipelines-shortname} supports inline specification of pipelines and tasks in the following cases:
```

**Comment by adellape** [comment]
@mramendi I see it looks we got QE review here inc oments and the issue was VERIFIED, but just a reminder to check the "QE has approved this change" box in PR description to save some merge-review time. 🙇

**Pull request merged**